Makefile

```diff
@@ -97,7 +97,6 @@ multimod-verify:
 .PHONY: multimod-prerelease
 multimod-prerelease:
 	$(MULTIMOD) prerelease -s=true -b=false -v ./versions.yaml -m liatrio-otel
-	$(MAKE) tidy-all
```

Comment:
Why did we remove tidy-all?

Reply:
➜ multimod git:(main) ./multimod prerelease -h
Updates version numbers and commits to a new branch for release:

Checks that the working tree is clean.
Checks that Git tags do not already exist for the new module set version.
Switches to a new branch called prerelease__.
Updates version.go files, if they exist.
Updates module versions in all go.mod files.
Attempts to call 'go mod tidy' in the directory of each modified go.mod file.
Adds and commits changes to Git branch

-----

I removed tidy-all because I believed that prerelease already called tidy, but I can see it only calls it in the go.mod directories

Reply:
yea, i think we should probably keep tidy-all.